dkg/frostp2p.go

```diff
@@ -70,7 +76,45 @@ func newFrostP2P(tcpNode host.Host, peers map[peer.ID]cluster.NodeIdx, bcastComp
 		round1CastsRecv: round1CastsRecv,
 		round1P2PRecv:   round1P2PRecv,
 		round2CastsRecv: round2CastsRecv,
+	}, nil
+}
+
+// newCheckMsg returns a bcast.CheckMessage function for round 1 and 2.
+func newCheckMsg(messageID string) (bcast.CheckMessage, error) {
```

Comment:
so here are we checking if the message is of the correct format to validate?

Reply:
Correct.